### PR TITLE
Setup Travis CI for AWS EB

### PR DESCRIPTION
**Pull request by rhernand3z**
- Organized .gitignore and added more files/folders
- Added necessary Travis configuration in `.travis.yml`
- Updated README

### CODE REVIEW

**Comment by rhernand3z** [comment]
Tested this on QA prior to sending deploy target branch to master

http://new-arrivals-qa.us-east-1.elasticbeanstalk.com/

Running AWS QA version: 
`travis-59b61f404383bf3c96f2775ef115ca957b34f551-1515616170`

**Comment by rhernand3z** [comment]
Let's review this so we can close and show sys engineering...

CC: @kfriedman @EdwinGuzman

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+script: npm run dist
+before_deploy: "echo 'Successfully built distribution assets for New Arrivals; Preparing to deploy New Arrivals to AWS'"
+deploy:
+  skip_cleanup: true
+  provider: elasticbeanstalk
+  access_key_id: $AWS_PRODUCTION_ACCESS_KEY_ID
+  secret_access_key: "$AWS_PRODUCTION_ACCESS_SECRET"
+  region: us-east-1
+  app: $AWS_APP_NAME
+  env: $AWS_ENVIRONMENT_NAME
```
Comment (kfriedman) [inline .travis.yml]: Hmm... just thinking out loud but should this be specified in the Travis environment or somehow derived from the particular branch that was deployed using `TRAVIS_BRANCH`? 

Maybe this is fine for a first version!

**Comment by kfriedman** [comment]
Looks perfect to me! 💯 

Really nice work.

**Pull request merged**